contracts/colonyNetwork/ColonyNetworkStorage.sol

```diff
@@ -89,16 +89,14 @@ contract ColonyNetworkStorage is CommonStorage, ColonyNetworkDataTypes, DSMath {
   mapping (address => uint256) pendingMiningRewards; // Storage slot 34
 
   uint256 totalMinerRewardPerCycle; // Storage slot 35
-  uint256 annualMetaColonyStipend; // Storage slot 36
-  uint256 lastMetaColonyStipendIssued; // Storage slot 37
```

Comment:
We need to leave these slots in, unfortunately, because this has already been deployed to goerli. We can repurpose them in the future though.